package com.rsupport.mobile1.test.common

object ErrorMessages {
    const val HTTP_EXCEPTION = "네트워크가 오류가 발생했습니다. 잠시 후 다시 시도해주세요"
    const val IO_EXCEPTION = "서버와 연결이 끊어졌습니다. 잠시 후 다시 시도해주세요"
    const val UNKNOWN_EXCEPTION = "오류가 발생했습니다. 잠시 후 다시 시도해주세요"
}